libs/cdk/src/lib/utils/directives/selectable-list/selection.service.spec.ts

```diff
@@ -36,7 +36,7 @@ class TestDirective implements SelectableItemToken<HTMLElement, string> {
         this.clicked.emit();
     }
 
-    elementRef(): ElementRef<HTMLElement> {
+    get elementRef(): ElementRef<HTMLElement> {
```

Comment:
I am noticing here and in a handful of components that they are still using private element refs and public getter functions. Why not use public elementref on these?

Reply:
@mikerodonnell89 in some cases elementRef is refefred to inner viewChild